site/content/docs/commands/secret-init.en.md

```diff
@@ -0,0 +1,94 @@
+                                Mutually exclusive with cli-input-yaml flag.
+      --overwrite               Optional. Whether to overwrite an existing secret.
+      --values stringToString   Values of the secret in each environment. Specified as <environment>=<value> separated by commas.
+                                Mutually exclusive with cli-input-yaml flag. (default [])
+```
+## How can I use it?
+Create a secret with prompts. You will be prompted for the name of the secret, and its values in each of your existing environments.
+```
+$ copilot secret init
+```
+
+Create a secret named `db_password` in multiple environments. You will be prompted for the `db_password`'s values you want for each of your existing environments.
+```
+$ copilot secret init --name db_password
+```
+Create secrets from `input.yml`. For the format of the YAML file please see <a href="#secret-init-cli-input-yaml">below</a>.
+```
+$ copilot secret init --cli-input-yaml input.yml
+```
+
+!!!info
+    It is recommended that you specify your secret's values through our prompts (e.g. by running `copilot secret init --name`) or from an input file by using the `--cli-input-yaml` flag. While the `--values` flag is a convenient way to specify secret values, your input may appear in your shell history as plaintext.
+
+## What's next?
+
+Copilot will create SSM parameters named `/copilot/<app name>/<env name>/secrets/<secret name>`. 
+Using the parameter names, you can then modify the `secrets` section in your [service's](https://aws.github.io/copilot-cli/docs/manifest/backend-service/#secrets) or [job's](https://aws.github.io/copilot-cli/docs/manifest/scheduled-job/#secrets) manifest to reference the secrets that are just created. 
+
+For example, suppose you have an application `my-app`, and you've created a secret `db_host` in `prod` and `dev` environments.
+You can modify your service's manifest as follows:
+```yaml
+environments:
+    prod:
+      secrets: 
+        DB_PASSWORD: /copilot/my-app/prod/secrets/db_password
+    dev:
+      secrets:
+        DB_PASSWORD: /copilot/my-app/dev/secrets/db_password
+```
+
+Once you deploy this updated manifest, your service or job will be able to access the environment variable `DB_PASSWORD`.
+It will have the value of the SSM parameter `/copilot/my-app/prod/secrets/db_password` if the service/job is deployed in a `prod` environment, and `/copilot/my-app/dev/secrets/db_password` if it's deployed in a `dev` environment.
+
+This works because ECS Agent will resolve the SSM parameter when it starts up your task, and set the environment variable for you.
+
+## <span id="secret-init-cli-input-yaml">How do I use the `--cli-input-yaml` flag?</span>
+You can specify multiple secrets, and their values in each of your existing environments in a file. Then you can use the file as the input to `--cli-input-yaml` flag. Copilot will read from the file and create or update the secrets accordingly.
```

Comment:
```suggestion
You can specify multiple secrets and their values in each of your existing environments in a file. Then you can use the file as the input to `--cli-input-yaml` flag. Copilot will read from the file and create or update the secrets accordingly.
```

Reply:
!!! My editor told me I should add a comma. Bad editor